.github/workflows/typescript_tests.yml

```
@@ -90,7 +103,10 @@ jobs:
       - name: Install Node
         uses: actions/setup-node@v4
         with:
-          node-version: '16'
+          node-version: '18'
+      - name: Install Develocity npm Agent
+        run: npm exec -y -- pacote extract @gradle-tech/develocity-agent ~/.node_libraries/@gradle-tech/develocity-agent@1.0.2
```

@clayburn The version is in the wrong place. First argument for the extract is the package, the second argument is the target directory

```suggestion
        run: npm exec -y -- pacote extract @gradle-tech/develocity-agent@1.0.2 ~/.node_libraries/@gradle-tech/develocity-agent
```